docs/import.md

```diff
@@ -1,52 +1,113 @@
-# Import
 
-GGUF models and select Safetensors models can be imported directly into Ollama.
+# Importing a model
 
-## Import GGUF
+You can import a model or fine tuned adapter into Ollama:
```

Comment:
Let's add a ToC at the top here with three links the headers so people can jump to the right one:

* Import from GGUF
* Import from Safetensors
* Import from Safetensors Adapter
* Sharing models on ollama.com